app/assets/javascripts/pages/EligibilityEstimatorController.js.coffee

```diff
@@ -2,20 +2,24 @@
 ###################################### CONTROLLER ##########################################
 ############################################################################################
 
-EligibilityEstimatorController = ($scope, $state) ->
-  formDefaults =
-    'household_size': ''
-    'income_timeframe': ''
-    'income_total': ''
+EligibilityEstimatorController = ($scope, $state, ListingService, IncomeCalculatorService) ->
+  $scope.filter_defaults = ListingService.eligibility_filter_defaults
+  $scope.filters = angular.copy(ListingService.eligibility_filters)
+
+  # check if we've used the IncomeCalculator to calculate income
+  if IncomeCalculatorService.calculateTotalYearlyIncome() > 0
```

Comment:
@chrisdolendo any lines there even setting `$scope.foo = bar` gets evaluated when the controller first runs, which in our case using the ui-router means when you reach the page that uses that controller. You could think of it like an initializer. So the `if` statement here is saying, "when we get to the eligibility estimator page, check if this condition is true, if so initialize the values on the page for income_total and income_timeframe"